archiver_test.go

```diff
@@ -0,0 +1,128 @@
+package debos_test
+
+import (
+	"github.com/go-debos/debos"
+	"github.com/stretchr/testify/assert"
+	"reflect"
+	"strings"
+	"testing"
+)
+
+func TestTar_default(t *testing.T) {
+
+	// New archive
+	// Expect Tar by default
+	archive, err := debos.NewArchive("test.tar.gz")
+	assert.NotEmpty(t, archive)
+	assert.Empty(t, err)
+
+	// Type must be Tar by default
+	arcType := archive.GetType()
+	assert.Equal(t, debos.Tar, arcType)
```

Comment:
May be useful to also do a typeassertion here

Reply:
Not need due rework. Check Archive structure.